### PR TITLE
Add extension for skipping sad songs

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

**Review by 3raxton — CHANGES_REQUESTED** [review]
Hey @JakeRoggenbuck, thanks for the PR; it helped me make the repo a bit more organized for future contributors and users of Spicetify searching for custom apps. 

That being said, these changes require a few things from you. Would you please place your custom app in a folder along with a README file detailing what it does and at least one screenshot of what it looks like when working? You can find more details on the structure [here.](https://github.com/3raxton/spicetify-custom-apps/blob/main/CONTRIBUTING.md#contributing-your-custom-apps-to-this-repo-follow-the-instructions-below)

Additionally, If you'd also like to add your name, a link to your repo, website, etc., and a link to your custom app repo, you can do so in the [AUTHORS.md file](https://github.com/3raxton/spicetify-custom-apps/blob/main/AUTHORS.md). It's not required, but I figured it might be nice to share attributions to custom app authors. 

Thanks for helping kickstart this project. I'm glad to have you as part of the repo! Let me know if you have any questions.

**Comment by JakeRoggenbuck** [comment]
@3raxton Yeah, sure thing! I added the stuff in listed in the CONTRIBUTING.md. Thanks so much!

**Comment by 3raxton** [comment]
This looks good! There may be a problem with the picture, but I can't be sure until I merge. I'll keep you updated. Also, to confirm, you don't want to add anything to AUTHORS.md?

**Comment by JakeRoggenbuck** [comment]
Thanks! I added my extension and name to the AUTHORS.md.

File: AUTHORS.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Below are the authors who have shared their custom apps in this repo. Follow the
      - Spicetify extension to hide podcasts. Hides the podcasts link in the sidebar and podcast carousel items (e.g. in 'Home' and 'Made For You'). 
      - Based on the [ChristianSpotify](https://github.com/khanhas/spicetify-cli/wiki/Extensions#christian-spotify) extension and licensed under the [GPL-3.0 License](spicetify-hide-podcasts/LICENSE)
 
+### [@JakeRoggenbuck](https://github.com/JakeRoggenbuck)
+
+- [ auto-skip-sad-songs](https://github.com/JakeRoggenbuck/spicetify-cli-extensions)
+     -  Add a list of artists and songs to this extension, and toggle an auto skip for them
+
+
```
Comment (3raxton) [inline AUTHORS.md:39]: Would you mind removing this line?

Comment (3raxton) [reply AUTHORS.md:39]: Edited for claity

**Comment by 3raxton** [comment]
All is looking good! Thanks for keeping this in line with the current files' look; I appreciate it as tedious as it may have been. Glad to have you as part of the repo; this PR is merged!

**Comment by JakeRoggenbuck** [comment]
Yay! Thank you!

**Comment by 3raxton** [comment]
One thing to note here, the screenshot is not hosted correctly. I believe the adequate approach is hosting it in your forked repo and then committing it as another PR. Would you mind doing that?

**Comment by 3raxton** [comment]
@JakeRoggenbuck

**Comment by JakeRoggenbuck** [comment]
Sure, I can do that!

**Comment by JakeRoggenbuck** [comment]
Done in #6

**Comment by 3raxton** [comment]
Good stuff, appreciate the fast response!

**Pull request merged**